plugins/modules/platform_update.py

```diff
@@ -76,14 +76,14 @@
                             - 'C(Update): Performs an update.'
                             - 'C(Upgrade): Performs an upgrade.'
                             - When set to C(Update) or C(Upgrade), the C(sriov_adapter_update) will be implicit.
-                            - When set to C(NoUpdate), C(resource_type), and C(level) are not required.
+                            - When set to C(NoUpdate), C(repository), and C(level) are not required.
```

Comment:
Can you rephrase this line, its not clear
When the C(NoUpdate) parameter is set, the C(repository) and C(level) parameters are not required.